tests/controllers/qos_controller_test.cpp

```diff
@@ -37,8 +41,13 @@ TEST(SerenityControllerTest, NoQoSCorrections) {
   Try<Nothing> initialize = controller->initialize(
       lambda::bind(&MockSlaveUsage::usage, &usage));
 
+  process::Clock::pause();
```

Comment:
Hmm, i aware how to use this EXPECT_CALL, however here i just cannot use it because i am not expecting this future to be satisfied (: Also, i don't think we should spend too much time on this, because it is only temporary change.. We will introduce full pipeline soon, and it will fill that future properly, and the tests will change ((: